apps/studio/public/assets/css/preview-tw.css

```diff
@@ -2082,6 +2082,10 @@ video {
   list-style-type: none;
 }
 
+.auto-rows-max {
```

Comment:
the tests specifically mention safari 17. is that the minimum version we support?

Reply:
weirdly, it only affects safari 15, 16, 17 and not the earlier versions (<14). But yes, it works for all 3 of them